src/components/filter/expert/expertFilterConstants.ts

```diff
@@ -851,13 +851,6 @@ export const FIELDS_OPTIONS = {
         valueEditorType: 'select',
         defaultValue: '',
     },
-    SUBSTATION_PROPERTY_3: {
-        name: FieldType.SUBSTATION_PROPERTY_3,
-        label: 'substationProperty3Twt',
```

Comment:
I am quite sure that you can remove the associated loc : 
* substationProperty3Twt: 'Substation property 3',
* substationProperty3Twt: 'Propriété site 3',

Reply:
in both translations

Reply:
done